### DISCUSSION TITLE
On the mapping of the adenosine molecule

### DISCUSSION THREAD

**Discussion by yuqingxiong96** [ricalessandri/Martini3-small-molecules#34] [Q&A] [answered]
Hi, I want to use the Martini3 model to find the binding pocket of the protein. And the molecule is similar with the adenosine. So I am trying to map the adenosine. I read the itp fire for the adenosine from models/martini_v3.0.0_small_molecules_2020PCTSouzaNatCommun.itp and the Supplementary Information for "Protein-ligand binding with the coarse-grained Martini model". I want to know if TN3a and TN5a refer to the two N and one C on the adenine?  I know TC6 as the virtual site because that tow rings be modeled with a "hinge" construction, but I'm confused about the TN5a as the virtual site. And the mass of tiny and small beads is 36.79 and 60.05, how to calculate it?

**Comment by ricalessandri** [comment, marked as answer]
Hi @yuqingxiong96 . So, adenosine is mapped according to this scheme (I think this makes clear which atom belongs to which bead): https://github.com/ricalessandri/Martini3-mappings/blob/main/adenosine.pdf

And here's the topology:
https://github.com/ricalessandri/Martini3-small-molecules/blob/2113a594c2b52ce56dfc957d520d7718957b6107/collections/martini_v3.0.0_small_molecules_2020PCTSouzaNatCommun.itp#L250-L270

To your questions:
- **TN5a and TN3a mapping.** TN5a maps the C=N group; TN3a maps only the N (so, this molecule is a little "overmapped" if you consider only the aromatic ring).
- **TN5a bead as a virtual site.** This allows us to reduce the number of constrains in this model.
- **Masses.** In this case, we used the *actual* mass of adenosine (~267 g/mol) and each bead on the sugar is assigned the mass that it describes (for example, bead SP2 is O-CH-CH2-OH, which means mass 16+12+1+12+2+16+1=60, and similarly for SP3) while the mass of the ring is equally shared among all the non-virtual beads.

Let me know if this clears out the doubts or not.

**Reply by yuqingxiong96** [reply]
Thank you very much for your answer. I have understood how to calculate the mass of each bead. However, I have another question: Isn't the mass of beads determined according to the size of beads in martini3? Why do we use the mass of the atom in this calculation?

**Reply by ricalessandri** [reply]
Yes, you are correct. From the [Martini 3 paper supporting information](https://static-content.springer.com/esm/art%3A10.1038%2Fs41592-021-01098-3/MediaObjects/41592_2021_1098_MOESM1_ESM.pdf): "_Usually, Martini CG models use default size-dependent masses for the beads. In Martini 3, the default masses of R-, S- and T-beads are 72, 54, and 36 amu, respectively. Note that the dynamics of the system is affected by the mass assignment, so the correct ones (based on the sum of the masses of the atoms mapped by the bead) should be used in case trends in dynamic properties of the system are important._"
For the [ligands](https://github.com/ricalessandri/Martini3-small-molecules/blob/6640e443f7d826a3b6279a97c4ab045a34dde3d3/models/martini_v3.0.0_small_molecules_2020PCTSouzaNatCommun.itp) binding to the A2AR and FXR receptors, which were simulated in the [Martini 3 protein-ligand binding paper](https://doi.org/10.1038/s41467-020-17437-5), we chose to use the actual masses to preserve trends in dynamic properties.
